toolchain/templates/bridges2.mako

```diff
@@ -39,9 +39,9 @@ echo
     ${helpers.run_prologue(target)}
 
     % if not mpi:
-        (set -x; ${' '.join([f"'{x}'" for x in profiler ])} "${target.get_install_binpath(case)}")
```

Comment:
`set -x` is pretty useful. If this is only an issue on Frontier, it would be nice to keep it on other systems. There might also be a way of fixing the issue there, I have not looked into it.

Reply:
I mean, sure it might be useful, but if it's not necessary here and there's an alternative that's portable I think the portable alternative is more useful.

Reply:
Just to be clear, I was only talking about the `set -x` part. The rest of the changes are great. It is my understanding that `set -x` only causes trouble on Frontier, not any other system. Correct me if I'm wrong!

Reply:
Presumably we'll get access to more AMD machines in the future that will use the same profiling software as Frontier. I don't imagine AMD machines are going away any time soon.

Reply:
did @henryleberre and @wilfonba come to a consensus on this?

Reply:
Looks like @wilfonba added these back in, even to the frontier template.

Reply:
as long as it works...

Reply:
Yes. The problem was me copying the set with the profiler prepend and not leaving it at the beginning of of the command. It was a misunderstanding of what set was doing.